dependencies.gradle

```diff
@@ -3,7 +3,7 @@ ext.versions = [
 
         rxJava           : '1.3.0',
         jCommander       : '1.71',
-        commander        : '0.1.6',
+        commander        : '0.1.7',
```

Comment:
why in this pr tho? pls extract to separate one

Reply:
because it addresses the same thing: informative message on ANDROID_HOME error while running build, and no more